### ISSUE TITLE
start building out access:code database 

### ISSUE THREAD

**Issue by anandijain** [free2pee/Free2Pee.jl#8]
discussed w @michaelfortunato 

I think the most minimal table is 
1. osm node id (primary key)
2. code submission timestamp
3. code string 

we can keep it as a csv and just have it loaded in the Free2Pee module. then we can attempt a join on node id between `view_df` and our access code df on GET

**Comment by anandijain** [comment]
I think that this would be considered part of mvp since Ps notes app has this feature

**Comment by acoh64** [comment]
agreed. maybe add something for who submitted the code too

**Comment by anandijain** [comment]
yeah id like user_id too, we'll have to see if that requires us to figure out how to do the "sign in with OSM" thing. i don;t know how hard that will be

**Comment by anandijain** [comment]
ah @acoh64 the other nice thing about making PRs is that I can refer to them easily here. like #10 
but anyways, we started this in https://github.com/free2pee/Free2Pee.jl/commit/cda799e9d6648bd380bb5b169595962da3febef2 which just stores it in a dataframe that's hosted locally within the repo
